HW4/UniqueList/ListTests/ListTests.cs

```diff
@@ -0,0 +1,222 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+
+namespace UniqueListNumber
+{
+    [TestClass]
+    public class ListTests
+    {
+        private List list = new List();
```

Comment:
В C# 9 можно
```suggestion
        private List list = new();
```
и там вообще вроде как ни в каких ситуациях больше не надо писать тип дважды